admin/data/sql/migrations/088.do.create-job-table.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE [mtc_admin].[job] (
+  id        INT IDENTITY (1, 1) NOT NULL,
+  urlSlug [uniqueidentifier] NOT NULL DEFAULT (newid()),
+  input      [nvarchar](MAX) NOT NULL,
+  createdAt DATETIMEOFFSET(3)   NOT NULL DEFAULT GETUTCDATE(),
+  updatedAt DATETIMEOFFSET(3)   NOT NULL DEFAULT GETUTCDATE(),
+  version rowversion,
+  jobStatusCode char(3) NOT NULL,
+  jobTypeCode char(3) NOT NULL,
+  [output]    [nvarchar](MAX),
```

Comment:
Output is a reserved word in ANSI sql, ms sql server as well as ODBC.  I think we should change it.

Reply:
@jon-shipley changed to jobOutput